Assets/Scripts/StandardSamples/UI/MatchMaking/UISessionsMatchmakingMenu.cs

```diff
@@ -432,5 +366,45 @@ public void HideMenu()
 
             SessionsMatchmakingUIParent.gameObject.SetActive(false);
         }
+
+        public bool TryGetExistingUISessionEntryById(string sessionId, out UISessionEntry entry)
+        {
+            foreach (Transform childTransform in SessionContentParent.transform)
+            {
+                UISessionEntry thisEntry = childTransform.GetComponent<UISessionEntry>();
+
+                if (thisEntry == null)
+                {
+                    continue;
+                }
+
+                if (thisEntry.RepresentedSession == null || thisEntry.RepresentedSession.Id != sessionId)
+                {
+                    continue;
+                }
```

Comment:
Tiny nit would be to change this to:

```csharp
if (null == thisEntry || thisEntry.RepresentedSession == null || thisEntry.RepresentedSession.Id != sessionId)
{
    continue;
}
```

Reply:
Sure, can do! I have noticed that other programmers seem to prefer these kind of single-if statement checks, whereas I've preferred multiple statements. In this particular case, though, I think it really does make sense to do it the way you have.

Only question is, why put `null == thisEntry` instead of `thisEntry == null`? The order of comparison matters in PowerShell, but in C# it shouldn't.

Reply:
> Totally! So the reason I put null first, is because it's not possible to assign a value to null. In general, I put the immutable thing on the left-hand side. So it's the same for string literals and number values. The reason for all of this is because some languages allow assigning a value in a conditional, and if I do it the other way around, I might make the mistake of having the conditional look like this: if (thisEntry = null), which could be a difficult error to diagnose or hunt down. So for the sake of forming a habit that avoids that potential problem in any language, I put the literal value to the left.

(from conversation with Paul)

Reply:
We discussed this offline, but for the sake of the history: Putting the immutable value on the left-hand side of the conditional prevents accidental assignment happening in the conditional. Some languages allow for `thisEntry = null` inside the `if` statement. By forming a habit of putting the immutable thing first, I avoid making that mistake in languages that support doing so.